tests/rptest/tests/archival_test.py

```diff
@@ -34,6 +34,10 @@
 
 NTP = namedtuple("NTP", ['ns', 'topic', 'partition', 'revision'])
 
+# Log errors expected when connectivity between redpanda and the S3
+# backend is disrupted
+CONNECTION_ERROR_LOGS = ["archival - .*Failed to create archivers"]
```

Comment:
@Lazin please could you review this -- I've allow-listed this pattern because it's what the code currently outputs, but I wonder if the message could be improved to say something about not being able to reach the S3 endpoint (there are earlier lower-severity messages that hint at that, but a user might just filter for errors)